node_mentoring_t1_2_input_example.csv

```diff
@@ -0,0 +1,6 @@
+Book,Author,Amount,Price
```

Comment:
Consider separating your csv and text files by placing them in different folders.
E.x create a folders "csv" and "text" under "files" folder at the same root level as "src".

Also, consider creating a "core" folder, where you could store your project constants, like BASE_CSV_PATH = "../files/csv".

Reply:
Updated as per the comment